src/components/MapView/index.tsx

```diff
@@ -150,6 +150,22 @@ function MapView({ classes }: MapViewProps) {
     const hazardLayerId = urlParams.get('hazardLayerId');
     const baselineLayerId = urlParams.get('baselineLayerId');
 
+    /*
+      In case we don't have hazard or baseline layers we will use the default
+      layer provided in the appConfig defined within `prism.json` file.
+     */
+    if (!hazardLayerId && !baselineLayerId) {
+      const defaultLayer = get(appConfig, 'defaultLayer', '');
+      const layer = LayerDefinitions[defaultLayer as LayerKey];
```

Comment:
Would be nice if we showed a notification using the notification context if the defaultLayer param is invalid.

Reply:
Solved as well

```
if (Object.keys(LayerDefinitions).includes(defaultLayer)) {
        const layer = LayerDefinitions[defaultLayer as LayerKey];
        const urlLayerKey =
          layer.type === 'admin_level_data'
            ? BASELINE_LAYER_PARAM
            : HAZARD_LAYER_PARAM;
        updateHistory(urlLayerKey, defaultLayer);
      } else {
        dispatch(
          addNotification({
            message: `Invalid default layer identifier: ${defaultLayer}`,
            type: 'error',
          }),
        );
      }
```

Reply:
@erwagasore - the notification for an invalid default layer shows 5x. See screenshot below.

<img width="723" alt="Screen Shot 2021-11-30 at 18 04 05" src="https://user-images.githubusercontent.com/3343536/144170868-6f9905c4-38b4-4a26-9048-e3b8308e333e.png">

Reply:
Solved. Good catch @wadhwamatic. This was being caused by state change in `useEffect` added a local state as a boolean to only allow one attempt to set the default layer.